### PR TITLE
Update MongoDB dependency and use separated MongoDB URL and DB name

### PR DESCRIPTION
**Pull request by angellandros**
This PR should fix [issue 23](https://github.com/mikhail-angelov/mongo-unit/issues/23).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongo-unit",
-  "version": "1.4.5",
+  "version": "2.0.0",
```
Comment (angellandros) [inline package.json]: Breaking changes, right? Although, I have added backward compatibility.

Comment (mikhail-angelov) [reply package.json]: thank you for PR, I see mongo driver was updated
I updated the library and bumped dep versions

Comment (mikhail-angelov) [reply package.json]: V 1.5

**Comment by mikhail-angelov** [comment]
PR is cool, but I make a bit different update for this

**Pull request closed without merging**